### ISSUE TITLE
Suggestion: Allow zoom for the "goal" items

### ISSUE THREAD

**Issue by binarycow** [tobspr/shapez.io#15]
On the left side of the screen where the "goal" items are...  Sometimes, it's hard to make out the details to see exactly what the parts the item is constructed from.

Please allow a way to zoom in to see the items in more detail.

**Comment by tobspr** [comment]
You can increase the UI scale in the settings - that will also make them bigger

**Comment by bbugh** [comment]
It would be nice to show the current goal and reward on the Upgrades screen.

**Comment by binarycow** [comment]
This helps some, but for the complicated composite shapes (for example, 3 shapes stacked on top of each other) it's still hard to make out which "sub-shape" it is. 

Take, for example, this shape (currently viewed at interface scale "Large")
![image](https://user-images.githubusercontent.com/37249876/82902215-1f8e8500-9f2d-11ea-8f25-6ff3c3d8c1e2.png)

I needed to figure out which of these "sub-shapes" I needed:
![image](https://user-images.githubusercontent.com/37249876/82902171-10a7d280-9f2d-11ea-86df-9b0e97f9333f.png).  They are very similar, and when you stack shapes multiple times, they get smaller each time - losing detail.  I actually took a screenshot, pasted it into paint, and zoomed in via Paint, so I could make out the details.

I have a 22 inch monitor, 1920x1080.  Should I need to change the UI scale to Large or Huge to see the details?  

I think simply mousing over one of those goals should show a larger, zoomed-in version.
